E2E binary build is broken in master

Due to:

```
$ go build ./test/e2e
# github.com/kubernetes-incubator/service-catalog/test/e2e
../../go/src/github.com/kubernetes-incubator/service-catalog/test/e2e/util.go:37:13: undefined: brokerImageFlag
../../go/src/github.com/kubernetes-incubator/service-catalog/test/e2e/walkthrough.go:179:37: multiple-value util.WaitForBindingCondition() in single-value context
```

--- a/test/e2e/walkthrough.go
+++ b/test/e2e/walkthrough.go
@@ -176,7 +176,7 @@ var _ = framework.ServiceCatalogDescribe("walkthrough", func() {
 		Expect(binding).NotTo(BeNil())
 
 		By("Waiting for ServiceBinding to be ready")
-		err = util.WaitForBindingCondition(f.ServiceCatalogClientSet.ServicecatalogV1beta1(),
+		_, err = util.WaitForBindingCondition(f.ServiceCatalogClientSet.ServicecatalogV1beta1(),
 			testnamespace.Name,
 			bindingName,
 			v1beta1.ServiceBindingCondition{

